src/index_create_gcloud_datadog_role.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
Why do we need separate file?

Reply:
If in future we want to run datadog for already on boarded customers we can automate it and run it separately.